low-level/framework/sspl_ll_d

```diff
@@ -542,7 +544,7 @@ def _sensors_actuators_factory(sspl_threaded_modules, msgQlist, operating_system
                         % (module, threaded, klass.name()))
 
         # If it's threaded then add it to the list which will be handled by the ThreadController
-        if threaded in ['True', 'true']:
+        if threaded in ['True', 'true', True]:
```

Comment:
True is covered already right, whats the deal?

Reply:
Only string `'True'` is covered, conf store support boolean. Hence, used it.

Reply:
oh i missed that